preparer.py

```diff
@@ -51,9 +51,14 @@ def prepare(self, client, collection_path_guest, generate_save_tag):
                 "mode": "ro"
             }
 
+        init_args = {
+            "opts": {key: value for (key, value) in map(lambda x: x.split("="), self.config.opts)},
```

Comment:
It's fine for this PR, but at some point in the future we should probably make a function out of this and avoid copying the same code each time, especially if for example we decide to change how we parse the optional configurations.